src/test/java/org/folio/circulation/domain/ItemTest.java

```diff
@@ -0,0 +1,64 @@
+package org.folio.circulation.domain;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import java.util.UUID;
+
+import org.junit.Test;
+
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+
+public class ItemTest {
+
+  @Test
+  public void titleFromInstanceTakesPriorityOverItem() {
```

Comment:
Hmm, is there still code for handling a `title` property of an item?

Reply:
yes, there is getTitle method.

Reply:
Thanks. 

An item no longer has a `title`, please can either the production and test code for this be removed, or an additional technical debt story raised to do so.

Reply:
Hmm, the `getTitle` return title from instance representation. 
```
  public String getTitle() {
    if(instanceRepresentation != null
      && instanceRepresentation.containsKey(TITLE)) {

      return getProperty(instanceRepresentation, TITLE);
    }

    return getProperty(getItem(), TITLE);
  }
```

The method also used in production code.

Reply:
@bohdan-suprun Apologies for being unclear.

Yes, I mean removing all of the logic for using a `title` from an item record. The `instance` is the only record that now has a `title`.

Reply:
No prob, I have changed it.